console.rst

```diff
@@ -156,18 +156,16 @@ You can optionally define a description, help message and the
 
     // src/Command/CreateUserCommand.php
 
-    // ...
-    class CreateUserCommand extends Command
+    #[AsCommand(
+        name: 'app:create-user',
+        description: 'Creates a new user.', // the command description shown when running "php bin/console list"
+        help: 'This command allows you to create a user...', // the command help shown when running the command with the "--help" option
+    )]
+    class CreateUserCommand
     {
-        // ...
-        protected function configure(): void
+        public function __invoke(): int
         {
-            $this
-                // the command description shown when running "php bin/console list"
-                ->setDescription('Creates a new user.')
-                // the command help shown when running the command with the "--help" option
-                ->setHelp('This command allows you to create a user...')
-            ;
+            // ...
         }
     }
 
```

Comment:
The next section ("Registering the Command") also needs rework, the document now only shows the `#[AsCommand]` examples, so "You can register the command by adding the AsCommand attribute to it:" is out of place now.

However, we need to document the final paragraph ("If you can't use PHP attributes, register the command as a service [...]") somewhere in this article I think.